config.toml

```diff
@@ -0,0 +1,22 @@
+[mozci]
+verbose = 1
+data_sources = ['treeherder_db', 'hgmo', 'taskcluster', 'adr']
```

Comment:
What is adr?

Reply:
It's "active data recipes".  We shouldn't need it anymore, but the test fail without it.  So I was going to ping Marco about it because I think it's failing inside `mozci`.